README.md

```diff
@@ -152,6 +152,55 @@ using (IDbConnection conn = new SnowflakeDbConnection())
 }
 ```
 
+<a id="sample-connection-strings"></a>
+#### Using Special Characters in Connection Passwords
+
+The .NET connector now uses Microsoft [DbConnectionStringBuilder](https://learn.microsoft.com/en-us/dotnet/api/system.data.oledb.oledbconnection.connectionstring?view=dotnet-plat-ext-6.0#remarks) to following .NET specification for escaping characters in connection strings. 
```

Comment:
Should "to following .NET specification" be "to follow the .NET specification"?

Reply:
Done